Development/make_netCDF/Create_NetCDF_Files_V0.r

```diff
@@ -100,6 +100,49 @@ for (yr in yrs) {
           #Open the data. Current solution to "WRITE_BOTTOMUP" is 
           #to manually flip the image 
           mat <- matrix(as.numeric(readGDAL(paste0(inRast_Base,lyr$short_name,'.tif'),silent=T)$band1),length(x),length(y))
+          
+          # Where erroneous EVI values (i.e., EVImax >1,EVIamp >1, EVIarea >3700)
+          # assign NAs for NumCycles, phenometrics and EVI layers, and 5 for QA layers
+          matEVImax  <- matrix(as.numeric(readGDAL(paste0(inRast_Base,'EVImax.tif'),silent=T)$band1),length(x),length(y))
+          matEVIamp  <- matrix(as.numeric(readGDAL(paste0(inRast_Base,'EVIamp.tif'),silent=T)$band1),length(x),length(y))
+          matEVIarea <- matrix(as.numeric(readGDAL(paste0(inRast_Base,'EVIarea.tif'),silent=T)$band1),length(x),length(y))
+          matEVImax_2  <- matrix(as.numeric(readGDAL(paste0(inRast_Base,'EVImax_2.tif'),silent=T)$band1),length(x),length(y))
+          matEVIamp_2  <- matrix(as.numeric(readGDAL(paste0(inRast_Base,'EVIamp_2.tif'),silent=T)$band1),length(x),length(y))
+          matEVIarea_2 <- matrix(as.numeric(readGDAL(paste0(inRast_Base,'EVIarea_2.tif'),silent=T)$band1),length(x),length(y))
+          
+          if(i<20){ # NumCycles, phenometrics and EVI layers
+            mat[matEVImax  >10000] <- NA  
+            mat[matEVIamp  >10000] <- NA  
+            mat[matEVIarea >37000] <- NA  
```

Comment:
I think you should change 37000 to 32766 for eviArea, because 32767 is the fill value, so we should only consider values below the fill value. We should never see values this high anyway. Additionally, you would need to change the valid range to 0 - 32766 in the metadata